pilot/lib/pilot/options.rb

```diff
@@ -46,7 +46,10 @@ def self.available_options
                                      short_option: "-d",
                                      optional: true,
                                      env_name: "PILOT_BETA_APP_DESCRIPTION",
-                                     description: "Provide the beta app description when uploading a new build"),
+                                     description: "Provide the beta app description when uploading a new build",
+                                     verify_block: proc do |value|
+                                       UI.user_error!("Beta App Description must be at least 10 characters long.") unless options[:beta_app_description].to_s.length >= 10
```

Comment:
Can you check the `value.to_s.length` rather than `options[:beta_app_description]`?